PORTAL/jobs/__init__.py

```diff
@@ -103,6 +105,12 @@ def __init__(self,
     def ssh(self):
         return self.bench_ssh
 
+    @classmethod
+    def find_config(cls, worker=None, cfgdirs=None):
+        if worker is not None:
+            cfgdirs = [f"/home/benchmarking/BENCH/WORKERS/{worker}"]
+        return super().find_config(cfgdirs)
```

Comment:
We may want to re-think having a per-worker config on the portal.  I'd expect the portal to have a single config which holds the list of workers:

```json
{
    "local_user": "benchmarking",
    "workers": [
        "linux": {
            "user": "benchmarking",
            "ssh_host": "localhost",
            "ssh_port": 22222
        }
    ]
}
```

Then we wouldn't need to worry about the worker until we call `Jobs._get()`.  That means a variety of things get a little cleaner, including that `--worker` doesn't need to be a "common" CLI arg any more

----

FYI, I realized that the existing code wasn't exactly clear in this area.  I created #78 to help with that.

Reply:
It would probably be smart to record the worker's OS in the config too.

Reply:
Yeah, that's a good idea. That would remove the need for the `BENCH/WORKERS` directory on the portal as well, AFAICT.